### PR TITLE
Bugfix/small change to update user controller

### PR DESCRIPTION
**Pull request by nicopicchio**
Added profile image logic to user controller

### CODE REVIEW

**Comment by marianphillips** [comment]
why was there an update to prisma? is this necesaary?

**Review by JasonKundi — APPROVED** [review]
Looks like a simple PR. Inclusion of profile image into req.body on the back end so that should fix the problem of including a profile image. Nice work amigo

File: package.json [hunk 1]
```diff
@@ -32,10 +32,10 @@
     "husky": "^7.0.4",
     "nodemon": "^2.0.15",
     "prettier": "^2.6.2",
-    "prisma": "^3.12.0"
+    "prisma": "^3.14.0"
   },
   "dependencies": {
-    "@prisma/client": "^3.12.0",
+    "@prisma/client": "^3.14.0",
```
Comment (mikemherron) [inline package.json:38]: Is this required?

**Comment by nicopicchio** [comment]
@marianphillips because I updated it to the latest version.

**Review by mikemherron — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nicopicchio) [reply package.json:38]: not sure how to go back

**Pull request merged**